docs/source/notes/get_start_xpu.rst

```diff
@@ -0,0 +1,417 @@
+
+Pytorch 2.4: Getting Started on Intel GPU
+=========================================
+
+The support for Intel GPUs is released alongside PyTorch v2.4.
+
+This release only supports build from source for Intel GPUs.
+
+Hardware Prerequiste
+--------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Supported Hardware
+     - Intel® Data Center GPU Max Series
+   * - Supported OS
+     - Linux
+
+
+Pytorch for Intel GPUs is compatible with Intel® Data Center GPU Max Series and Intel® Core™ Ultra Processors, and only support OS Linux with release 2.4.
+
+Software Prerequiste
+--------------------
+
+Install driver and required packages by following the `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_ to Get Started.
+
+Set up Environment
+------------------
+
+Before you begin, you need to set up the environment. This can be done by sourcing the setvars.sh script provided by the ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages.
+
+.. code-block::
+
+   source ${ONEAPI_ROOT}/setvars.sh
```

Comment:
I think this also needs to be mentioned in top level readme: https://github.com/pytorch/pytorch?tab=readme-ov-file#intel-gpu-support

Reply:
Hi @rivergold , please help on update in top level readme.

Reply:
Hi @riverliuintel, please help update in top level readme